zetaclient/bitcoin/fee.go

```diff
@@ -0,0 +1,234 @@
+package bitcoin
+
+import (
+	"encoding/hex"
+	"fmt"
+	"math"
+	"math/big"
+
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/btcjson"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcutil"
+	"github.com/rs/zerolog"
+	"github.com/zeta-chain/zetacore/common"
+	"github.com/zeta-chain/zetacore/common/bitcoin"
+	clientcommon "github.com/zeta-chain/zetacore/zetaclient/common"
+
+	"github.com/btcsuite/btcd/wire"
+	"github.com/pkg/errors"
+)
+
+const (
+	bytesPerKB              = 1000
+	bytesEmptyTx            = 10  // an empty tx is 10 bytes
```

Comment:
Do you have a source where these numbers have been fetched?
It would be relevant to put this reference in this file as a comment

Reply:
I'll remove it as it's used nowhere now.

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/2bcdfbc727c12612159ffbf008bb00505a996b4b